portal/src/main/webapp/WEB-INF/jsp/global/frontend_config.jsp

```diff
@@ -98,10 +98,13 @@ if (/localdist=true/.test(window.location.href)) {
 }
 window.localdist = localStorage.getItem("localdist") === "true";
 
-if (window.localdist || window.localdev) {
-	window.frontendConfig.frontendUrl = '//localhost:3000/'
+if (window.localdist) {
+    window.frontendConfig.frontendUrl = '//localhost:3000/'
```

Comment:
do we still need ssl for `localdist`? I thought that one also always goes over ssl these days:

https://github.com/cBioPortal/cbioportal-frontend/blob/master/scripts/serve_dist.sh#L20-L27